<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>993</width>
        <height>557</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>邮箱</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0" colspan="2">
         <widget class="QProgressBar" name="em_progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>日志消息</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QTextBrowser" name="em_log_msg">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="em_user">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>发送者邮箱账号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="em_pwd">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>发送者邮箱密码</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="em_login">
            <property name="maximumSize">
             <size>
              <width>9999999</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>登录</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1" rowspan="3">
         <widget class="QTabWidget" name="em_function_zone">
          <property name="minimumSize">
           <size>
            <width>811</width>
            <height>475</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>邮件群发</string>
           </attribute>
           <widget class="QLineEdit" name="em_mass_recipients">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>671</width>
              <height>31</height>
             </rect>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>接收者邮箱：多个请使用“,”相隔</string>
            </property>
           </widget>
           <widget class="QTextEdit" name="em_mass_content">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>721</width>
              <height>191</height>
             </rect>
            </property>
            <property name="maximumSize">
             <size>
              <width>721</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>请在这里输入正文</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="em_mass_subject">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>50</y>
              <width>511</width>
              <height>31</height>
             </rect>
            </property>
            <property name="placeholderText">
             <string>请输入邮件主题</string>
            </property>
           </widget>
           <widget class="QPushButton" name="em_mass_send">
            <property name="geometry">
             <rect>
              <x>280</x>
              <y>360</y>
              <width>141</width>
              <height>61</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>立即发送</string>
            </property>
           </widget>
           <widget class="QPushButton" name="em_mass_add_attachment">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>280</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>添加附件</string>
            </property>
           </widget>
           <widget class="QPushButton" name="em_mass_import_file_content">
            <property name="geometry">
             <rect>
              <x>730</x>
              <y>90</y>
              <width>61</width>
              <height>191</height>
             </rect>
            </property>
            <property name="maximumSize">
             <size>
              <width>721</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <stylestrategy>PreferDefault</stylestrategy>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="text">
             <string>从
文
件
导
入
内
容</string>
            </property>
           </widget>
           <widget class="QPushButton" name="em_mass_import_file_recipients">
            <property name="geometry">
             <rect>
              <x>690</x>
              <y>10</y>
              <width>111</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>从文件导入</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="em_mass_text">
            <property name="geometry">
             <rect>
              <x>230</x>
              <y>330</y>
              <width>115</width>
              <height>19</height>
             </rect>
            </property>
            <property name="text">
             <string>text</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>310</y>
              <width>72</width>
              <height>15</height>
             </rect>
            </property>
            <property name="text">
             <string>文本类型</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="em_mass_html">
            <property name="geometry">
             <rect>
              <x>390</x>
              <y>330</y>
              <width>115</width>
              <height>19</height>
             </rect>
            </property>
            <property name="text">
             <string>html</string>
            </property>
           </widget>
           <widget class="QPushButton" name="em_mass_clock_send">
            <property name="geometry">
             <rect>
              <x>590</x>
              <y>320</y>
              <width>181</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>开启定时发送</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="em_mass_evday">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>640</x>
              <y>410</y>
              <width>131</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>每天发送</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>590</x>
              <y>380</y>
              <width>179</width>
              <height>27</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string>定时时间</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTimeEdit" name="em_mass_clock"/>
             </item>
            </layout>
           </widget>
           <widget class="QListWidget" name="em_mass_attachment">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>300</y>
              <width>211</width>
              <height>171</height>
             </rect>
            </property>
           </widget>
           <zorder>em_mass_clock_send</zorder>
           <zorder>em_mass_evday</zorder>
           <zorder>layoutWidget</zorder>
           <zorder>em_mass_recipients</zorder>
           <zorder>em_mass_content</zorder>
           <zorder>em_mass_subject</zorder>
           <zorder>em_mass_send</zorder>
           <zorder>em_mass_add_attachment</zorder>
           <zorder>em_mass_import_file_content</zorder>
           <zorder>em_mass_import_file_recipients</zorder>
           <zorder>em_mass_text</zorder>
           <zorder>label_4</zorder>
           <zorder>em_mass_html</zorder>
           <zorder>em_mass_attachment</zorder>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>自动回复</string>
           </attribute>
           <widget class="QPushButton" name="em_auto_start_reply">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>370</y>
              <width>191</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>开启自动回复</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="em_auto_text">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>320</y>
              <width>115</width>
              <height>19</height>
             </rect>
            </property>
            <property name="text">
             <string>text</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QTextEdit" name="em_auto_content">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>721</width>
              <height>191</height>
             </rect>
            </property>
            <property name="maximumSize">
             <size>
              <width>721</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="placeholderText">
             <string>请在这里输入正文</string>
            </property>
           </widget>
           <widget class="QPushButton" name="em_auto_import_file_content">
            <property name="geometry">
             <rect>
              <x>730</x>
              <y>90</y>
              <width>61</width>
              <height>191</height>
             </rect>
            </property>
            <property name="maximumSize">
             <size>
              <width>721</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <stylestrategy>PreferDefault</stylestrategy>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="text">
             <string>从
文
件
导
入
内
容</string>
            </property>
           </widget>
           <widget class="QPushButton" name="em_auto_add_attachment">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>280</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>添加附件</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="em_auto_html">
            <property name="geometry">
             <rect>
              <x>400</x>
              <y>320</y>
              <width>115</width>
              <height>19</height>
             </rect>
            </property>
            <property name="text">
             <string>html</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="em_auto_subject">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>50</y>
              <width>511</width>
              <height>31</height>
             </rect>
            </property>
            <property name="placeholderText">
             <string>请输入邮件主题</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>320</x>
              <y>300</y>
              <width>72</width>
              <height>15</height>
             </rect>
            </property>
            <property name="text">
             <string>文本类型</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>10</y>
              <width>791</width>
              <height>41</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="em_auto_import_file_recipient">
               <property name="text">
                <string>从文件导入</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="em_auto_recipients">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>31</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>对特定人回复：多个请使用“,”相隔</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="em_auto_reply_all">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>对所有人回复</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QListWidget" name="em_auto_attachment">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>300</y>
              <width>211</width>
              <height>171</height>
             </rect>
            </property>
           </widget>
           <widget class="QListView" name="listView">
            <property name="geometry">
             <rect>
              <x>1000</x>
              <y>160</y>
              <width>256</width>
              <height>192</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>扩展功能</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>qq</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>微信</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>短信</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionFile"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>代码</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
